src/dbcp/validation/tests.py

```diff
@@ -240,7 +241,7 @@ def test_county_wide_coverage(engine: Engine):
     ), "counties_wide_format does not contain all counties"
     notnull = df.notnull()
     assert (
-        notnull.any(axis=1).sum() == 2387
+        notnull.any(axis=1).sum() == 2423
```

Comment:
I ran this locally and it wanted 2424 places. But it could be a cache thing. I guess the GH Action will check for us (it has no cache, right?)